.github/workflows/pyinstaller-windows.yml

```diff
@@ -0,0 +1,26 @@
+name: Package Application with Pyinstaller
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v2
+
+    - name: Package Application
+      uses: JackMcKew/pyinstaller-action-windows@main
+      with:
+        path: mathlibtools
+        spec: leanproject.spec
+
+    - uses: actions/upload-artifact@v2
+      with:
+        name: leanproject.exe
```

Comment:
Perhaps it's reasonable enough here to add a second step that uses the artifact that was built in this first step to do a simple smoke test on a Windows machine?

I.e. just run `leanproject.exe --help` or something?

Reply:
Ah, the obvious problem here is that we're in a linux container!